1.8/administration/haproxy-adminrouter.md

```diff
@@ -10,6 +10,12 @@ The HTTP Proxy must perform on-the-fly HTTP request and response header modifica
 
 These instructions provide a tested [HAProxy](http://www.haproxy.org/) configuration example that handles the named request/response rewriting. This example ensures that the communication between HAProxy and DC/OS Admin Router is TLS-encrypted.
 
+**Tip:**
+
+During DNS lookup for a task’s IP, there are situations when it is unknown to the DNS server if the IP address stored in NetworkInfo is accessible or not. You can use these entries in Virtual Network Service DNS to find your task IP:
```

Comment:
Change to "To avoid having to reconfigure DC/OS, You can use this DNS entry to find your task IP."